test/e2e/scalability/density.go

```diff
@@ -171,9 +171,9 @@ func density30AddonResourceVerifier(numNodes int) map[string]framework.ResourceC
 		}
 	} else {
 		if numNodes <= 100 {
-			apiserverCPU = 1.8
+			apiserverCPU = 2.2
 			apiserverMem = 1700 * (1024 * 1024)
-			controllerCPU = 0.6
+			controllerCPU = 0.9
```

Comment:
I think 0.8 would be better (0.9 is giving more slack than required) - unless there's some counter example

Reply:
Fixed.